tests/e2e/project/settings/slr/fetcher-options.test.ts

```diff
@@ -0,0 +1,72 @@
+import { expect } from "@playwright/test";
+import { reloadWait } from "$tests/e2e/utils/helper/helper";
+import { test } from "./project-slr-settings-page-fixture";
+
+test.describe("SLR Settings Fetcher Options", () => {
+    test("When a project has a fetcher, then it is visible", async ({
+        projectSLRSettingsPage: slrPage,
+    }) => {
+        await slrPage.insertFetcher("test");
+        await reloadWait(
+            slrPage.page,
+            slrPage.page.getByRole("heading", { name: "test", exact: true }),
+        );
+    });
+
+    test("When the user adds a fetcher, then it is added and visible", async ({
+        projectSLRSettingsPage: slrPage,
+    }) => {
+        await slrPage.addFetcher("test");
+        await expect(slrPage.getFetcherTitle("test")).toBeVisible();
+    });
+
+    test("When the user deletes a fetcher, then it is not visible anymore", async ({
+        projectSLRSettingsPage: slrPage,
+    }) => {
+        await slrPage.insertFetcher("test");
+        await reloadWait(slrPage.page, slrPage.getFetcherTitle("test"));
+        await slrPage.deleteFetcher("test");
+        await expect(slrPage.getFetcherRow("test")).not.toBeVisible();
+    });
+
+    test("When the user edits a fetcher, then it is saved across reloads", async ({
+        projectSLRSettingsPage: slrPage,
+    }) => {
+        await slrPage.insertFetcher("test");
+        await reloadWait(slrPage.page, slrPage.getFetcherTitle("test"));
+        await slrPage.openEditFetcherDialog("test");
+        await slrPage.page.getByRole("textbox").nth(1).fill("foobar");
+        await slrPage.page.getByTestId("alert-dialog-action").click();
+        await reloadWait(slrPage.page, slrPage.getFetcherTitle("test"));
+        await slrPage.openEditFetcherDialog("test");
+        await expect(slrPage.page.getByRole("textbox").nth(1)).toHaveValue("foobar");
+    });
+
+    test("When the user inputs a fetcher option value, then it is marked as overridden", async ({
+        projectSLRSettingsPage: slrPage,
+    }) => {
+        await slrPage.insertFetcher("test");
+        await reloadWait(slrPage.page, slrPage.getFetcherTitle("test"));
+        await slrPage.openEditFetcherDialog("test");
+        await expect(slrPage.page.getByRole("checkbox").nth(0)).not.toBeChecked();
```

Comment:
Use `first()` instead of `nth(0)`. Please correct this every time you use `nth(0)` here and in `project-slr-settings-page-model.ts` as well.
```suggestion
        await slrPage.page.getByRole("textbox").first().fill("foobar");
```